src/main.cpp

```diff
@@ -256,8 +256,10 @@ int main(int argc, char *argv[]){
 
         std::cout << "\n⭐ CPU analysis ⭐" << std::endl;
         Stopwatch cpu_watch;
+        auto init_task = init_gpu_async();
```

Comment:
shouldnt the gpu init be done when the GPU part is run and not when the CPU analysis is being run?